github/client.go

```diff
@@ -117,21 +117,34 @@ var reviewLookupTable = map[string]Review{
 
 // PullRequestReviews returns a slice of review states on the pull request.
 func (r RepositoryClient) PullRequestReviews(number int) ([]Review, error) {
-	// TODO: paging might be an issue if there are a lot of reviews.
-	reviews, _, err := r.client.PullRequests.ListReviews(context.TODO(), r.owner, r.name, number, nil)
-	if err != nil {
-		return nil, fmt.Errorf("failed to list reviews for pull request: %w", err)
+	opt := &github.ListOptions{PerPage: 100}
+	var allReviews []*github.PullRequestReview
+	for {
+		reviews, resp, err := r.client.PullRequests.ListReviews(context.TODO(), r.owner, r.name, number, opt)
```

Comment:
What does `number` represent?

Reply:
The "number" of the pull request. It isn't an ID, because reviews have global ids called id as well. I decided to match it to the nomenclature of the Github API.